__tests__/helper.test.ts

```diff
@@ -33,8 +35,63 @@ function mockFetch(status: number, data?: any) {
   }, 0);
 }
 
+function getMetadataValue(MetadataGroups: any[], metaLabel: string){
+  let metaValue = "";
+  MetadataGroups.forEach(group => {
+    group.items.forEach(item => {
+      const labelText =
+        item.label?.getValue?.("en-GB") ??  
+        item.label?.getValue?.() ??
+        item.label?.toString?.();
+      if (labelText && labelText.trim().toLowerCase() === metaLabel) {
+        metaValue =
+          item.value?.getValue?.("en-GB") ??
+          item.value?.getValue?.() ??
+          item.value?.toString?.();
+      }
+    });
+  });
+  return (metaValue);
+}
+
 describe('Helper', () => {
 
+  test('getMetadata records summary as description when duplicate value present in metadata for brideMatch manifest', async () => {
+    const helper = await loadManifestJson(brideMatch, {
+      manifestUri: brideMatch.id, locale: 'cy'
+    });
+    const MetadataGroups = helper.getMetadata();
+    let check = false;
+    let summary = getMetadataValue(MetadataGroups,'summary');
+    let description = getMetadataValue(MetadataGroups,'description');
+    if(description != "" && summary === ""){check = true};
+    expect(check).toBe(true);
```

Comment:
I would suggest more specific expectations here -- rather than creating one boolean and testing that, I'd test each condition independently. That way, if something goes wrong, the failure message will be more detailed and helpful. There's nothing wrong with a single test making many assertions.

```suggestion
    expect(description).toEqual("<p><p>OC PS2394 .M642 1883 [xiv], 144, 126, [4] p. ; 18 cm. First set of unnumbered pages are a series list; second set are a publisher's catalog.</p></p>");
    expect(summary).toEqual("");
```

(And if it's unwieldy to use that whole HTML paragraph in your expectations, you could modify the fixture with something shorter -- since the whole purpose is testing, it doesn't really matter what value is in there).